modules/nw-sriov-configuring-device.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="configuring-sr-iov-devices_{context}"]
+= Configuring SR-IOV network devices
+
+The SR-IOV Network Operator adds the `SriovNetworkNodePolicy.sriovnetwork.openshift.io` Custom Resource Definition (CRD) to {product-title}.
+You can configure the SR-IOV network device by creating a SriovNetworkNodePolicy Custom Resource (CR).
+
+[NOTE]
+=====
+When applying the configuration specified in a SriovNetworkNodePolicy CR, the SR-IOV Operator may drain the nodes, and in some cases, reboot nodes.
+It may take several minutes for a configuration change to apply.
+After the configuration update is applied, all the Pods in `sriov-network-operator` namespace will change to a `Running` status.
+=====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges.
+* You must have installed the SR-IOV Operator.
+
+.Procedure
+
+. Create the following SriovNetworkNodePolicy CR, and then save the YAML in the `<name>-sriov-node-network.yaml` file. Replace `<name>` with the name for this configuration.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: <name> <1>
+  namespace: sriov-network-operator <2>
+spec:
+  resourceName: <sriov_resource_name> <3>
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true" <4>
+  priority: <priority> <5>
+  mtu: <mtu> <6>
+  numVfs: <num> <7>
+  nicSelector: <8>
+    vendor: "<vendor_code>" <9>
+    deviceID: "<device_id>" <10>
+    pfName: ["<pf_name>", ...] <11>
+    rootDevices: ["<pci_bus_id>", "..."] <12>
+  deviceType: <device_type> <13>
+  isRdma: false <14>
+----
+<1> Specify a name for the CR.
+<2> Specify the namespace where the SR-IOV Operator is installed.
+<3> Specify the resource name of the SR-IOV device plug-in. The prefix `openshift.io/` will be added when it's referred in Pod spec. You can create multiple SriovNetworkNodePolicy CRs for a resource name.
+<4> Specify the node selector to select which node to be configured. User can choose to label the nodes manually or with tools like Kubernetes Node Feature Discovery.
+Only SR-IOV network devices on selected nodes will be configured. The SR-IOV
+CNI plug-in and device plug-in will be only deployed on selected nodes.
+<5> Specify an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`.
+<6> Specify a value for the maximum transmission unit (MTU) of the virtual function. The value for MUT must be in the range from `1` to `9000`. If you do not need to specify the MTU, specify a value of `''`. 
+<7> Specify the number of the virtual functions (VF) to create for the SR-IOV physical network device.
```

Comment:
@pliurh, what is _TotalVFs_ in this context? Is it a configuration parameter in the CR? Or some fixed value for Intel?

Reply:
TotalVFs is the maximum number VF can be configured. For Intel NICs, it’s a static value. For Mellanox NICs it is variable with max value 128 for the NICs we claim to support, so it can be changed by the sriov operator when user specify the `numVFs`.